src/cli/cli.coffee

```diff
@@ -23,7 +23,7 @@ cli.parse {
   "xsl-href": ['x', "path to xsl for include to xml report", "string"],
   "verbose": [false, "show full info about copies"]
   "debug": ['d', "show debug information(options list and selected files)"]
-  "path": ['p', "path to code", "path", process.cwd()]
```

Comment:
My aim was to take the path in priority, cmd line parameter, config file, process.cwd. With this default, we don't know whether the path was passed or just the fallback.